app/views/transaction/_additional_information_single.html.erb

```diff
@@ -1,21 +1,20 @@
-<% if presenter.before_you_start? %>
+<% if transaction.more_information.present? %>
   <div id="before-you-start">
-    <h2><%= t 'formats.transaction.before_you_start' %></h2>
-
+    <h2><%= t 'formats.transaction.more_information' %></h2>
     <%= raw transaction.more_information %>
   </div>
 <% end %>
 
-<% if presenter.what_you_need_to_know? %>
+<% if transaction.what_you_need_to_know.present? %>
   <div id="what-you-need-to-know">
     <h2><%= t 'formats.transaction.what_you_need_to_know' %></h2>
-    <%= render :partial => 'need_to_know', :locals => { :transaction => transaction } %>
+    <%= raw transaction.what_you_need_to_know %>
```

Comment:
@nickcolley - we'll be merging this soon, if you weren't happy with the changes let us know.